sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ImmutableStorageWithVersioningTests.java

```diff
@@ -610,10 +610,13 @@ private static Stream<Arguments> blobUploadSupplier() {
         return Stream.of(Arguments.of(1L), Arguments.of((Long) null));
     }
 
+    // Live mode does not support copy from URL with immutability policy due to public access
+    // Revisit this and remove public access once default credential is enabled
     @RequiredServiceVersion(clazz = BlobServiceVersion.class, min = "2020-10-02")
     @Test
+    @PlaybackOnly
     public void syncCopy() {
-        vlwContainer.setAccessPolicy(PublicAccessType.CONTAINER, null);
+        //vlwContainer.setAccessPolicy(PublicAccessType.CONTAINER, null);
         sleepIfRunningAgainstService(30000); // Give time for the policy to take effect
```

Comment:
totally forgot to write this - we can remove all of the sleep's as we aren't waiting for the access policy to set anymore